README.md

```diff
@@ -1,102 +1,90 @@
 # sui-card React Component
 
-The SUI-Card component features a generic div tag including two more nested containers one on top of the other.
+The SUI-Card component features a placeholder to portray a custom primary and a secondary content. So you can set an image, slider or any media component in one and some other content in the other.
 
 
 ## Demo page
 
-- Checkout a Sui-Card component demo [here](http://sui-components.github.io/sui-card/).
- <br /> or...
-- Clone the repo and open `index.html` file inside `docs` folder
-
+- Check out a Sui-Card component demo [here](http://sui-components.github.io/sui-card/).
 
 ## Usage
 
-The **SUI-Card** component features a generic `div` tag including two more nested containers one on top of the other.
-
-Top and bottom containers may fit any content provided to **Sui-Card**. Use the following classes to layout styling:
-
-- **sui-Card-topComponent**: <br />
-This is the top placeholder where you can fit any custom component: from a simple picture to a full featured slider.
-
-If no `topComponent` property is passed a default picture with link can be set using the following props:
+The **SUI-Card** allows to pass custom content in the primary placeholder and also provides a `sui-Card-primary` class as a wrapper.
 
-```
-// topImgDefault: Provide a custom image URL.
-// topImgDefaultLink: Provide a custom link.
-// topComponent: Provide any content as shown below:
-
-const top = (
-    <a href='http://bit.ly/1jt4zRp'>
-      <img src='http://bit.ly/1FzqHDX' />
-    </a>
-);
+Check out this example below
+```javascript
 
 ReactDom.render(
-  <SuiCard
-    topImgDefault='http://placehold.it/300x200'
-    topImgDefaultLink='http://google.com'
-
-    topComponent={top}
+    <SuiCard
+      primary={[
+        <SuiMultimedia data={data} />
+      ]}
+      secondary={[
```

Comment:
Agree.. sort of.. :)

The way `Sui-Card` is designed currently, `Media` can be applied inside `secondary` placeholder and text or just a button in the primary. This gives more flexibility to the developer because name space is not pre-setting any content.

I consider myself an open minded pal so please don't hesitate posting more naming proposals :)